File: app/src/main/java/com/example/dcct/bean/UserEntity.java
package com.example.dcct.bean;

public class UserEntity {
    private int uid;
    private Object password;
    private String nickname;
    private String email;

    @Override
    public String toString() {
        return "UserEntity{" +
                "uid=" + uid +
                ", password=" + password +
                ", nickname='" + nickname + '\'' +
                ", email='" + email + '\'' +
                '}';
    }

    public int getUid() {
        return uid;
    }

    public void setUid(int uid) {
        this.uid = uid;
    }

    public Object getPassword() {
        return password;
    }

    public void setPassword(Object password) {
        this.password = password;
    }

    public String getNickname() {
        return nickname;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

}

File: app/src/main/java/com/example/dcct/bean/UserEntity.kt
package com.example.dcct.bean

class UserEntity {
    var uid = 0
    var password: Any? = null
    var nickname: String? = null
    var email: String? = null
    override fun toString(): String {
        return "UserEntity{" +
                "uid=" + uid +
                ", password=" + password +
                ", nickname='" + nickname + '\'' +
                ", email='" + email + '\'' +
                '}'
    }

}